controllers/volumereplication_controller.go

```diff
@@ -156,12 +165,14 @@ func (r *VolumeReplicationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		}
 	} else {
 		if contains(instance.GetFinalizers(), volumeReplicationFinalizer) {
-			err := r.disableVolumeReplication(logger, volumeHandle, parameters, secret)
-			if err != nil {
-				logger.Error(err, "failed to disable replication")
-				return ctrl.Result{}, err
+			// continue to delete if operator is not able to retrieve PV
```

Comment:
`continue to delete if operator is not able to retrieve PV` should be `continue to delete if operator is able to retrieve PV`

Reply:
the comment is about delete not disable.

Reply:
added comment for disabling now. PTAL